src/Service/UserServiceInterface.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace OpenBibIdApi\Service;
+
+interface UserServiceInterface extends ServiceInterface
+{
+
+    /**
+     * Get the user info of the currently logged in user.
+     *
+     * @return \DOMDocument|null
+     *   A \DOMDocument containing the XML from the response, null if HTTP
+     *   status code 204 (No content) was returned.
+     *
+     * @throws BibApi\Exception\BibException
```

Comment:
@throws moet enkel toegevoegd worden in de documentatie als de methode zelf een exception throwed.